app/Http/Controllers/ReviewController.php

```diff
@@ -36,7 +36,27 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $review = new Review;
+
+        $review -> id;
+        $review -> store_id = $request -> store_id;
+        $review -> comment = $request -> comment;
+        $review -> created_at = $request -> created_at;
+        $review -> user_id = Auth::id();
+        $review -> review = $request -> review;
+        $review -> charge = $request -> charge;
+        $review -> ate_thing = $request -> ate_thing;
+        $review -> category_id = $request -> category_id;
+        $review -> post_image = $request -> post_image;
+        $review -> published_at = $request -> published_at;
```

Comment:
下書きか公開か、はviewから何かフラグをもらわないとですね！:eyes:
そのフラグによって時間を入れるか入れないか、にifで切り分けましょう！

あと `published_at ` はviewから送られてくるのではなくここで時間生成しましょうか！
```suggestion
        $review -> published_at = now();
```

Reply:
すみません、これは未対応です。